.circleci/config.yml

```diff
@@ -157,7 +157,9 @@ jobs:
 
 workflows:
   version: 2
-  ci-tests:
+  build-docs:
     jobs:
       - build-Docs
+  test-on-tpus:
```

Comment:
Note that they can currently be run separately inside the CircleCI website 
![Screenshot 2022-03-25 at 11 56 48](https://user-images.githubusercontent.com/11067892/160108349-82bcc9e1-8b12-4a09-9bd1-6f1ed5e92d8a.png)

Reply:
Can they really? I tried that before but both `TPU-tests` and `build-Docs` got rerun in the end...

Reply:
I think so, but this PR is still useful so we can do it from GitHub